Bookish/Views/Home/Index.cshtml

```diff
@@ -1,25 +1,37 @@
 ﻿@{
     ViewData["Title"] = "Home Page";
 }
-@using Bookish.Models
-@model BookInputModel
 
+@model BookOrBorrowerInputModel
 <div class="text-center">
     <h1 class="display-4">Welcome</h1>
     <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>

```

Comment:
You can delete this placeholder content :)

Reply:
This is fixed in Tom's branch but I've deleted it here, I believe this is what becomes Bookfish (and the fish picture as the subtitle) 🐟